### PR TITLE
Updates 'xeno queen' AI hologram

### PR DESCRIPTION
**Pull request by Spess-Rando**
:cl: Sligneris
imageadd: 'xeno queen' AI hologram now actually uses the xeno queen sprite as a reference
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 1]
```diff
@@ -697,14 +697,18 @@ var/list/ai_list = list()
...
 				"horror" = 'icons/mob/AI.dmi'
 				)
 
 			input = input("Please select a hologram:") as null|anything in icon_list
 			if(input)
 				qdel(holo_icon)
-				holo_icon = getHologramIcon(icon(icon_list[input], input))
+				switch(input)
+					if("xeno queen")
+						holo_icon = getHologramIcon(icon(icon_list[input],"alienq"))
```
Comment (ChangelingRain) [inline code/modules/mob/living/silicon/ai/ai.dm]: This does not automatically turn the xeno queen blue; you need to replace the existing icon with an updated blue version

**Comment by GunHog** [comment]
Does this use the LARGE alien queen?

File: code/modules/mob/living/silicon/ai/ai.dm (hunk 1 above)
Comment (Sligneris) [inline code/modules/mob/living/silicon/ai/ai.dm]: Strange, I see the exact same code on animal and crewmember versions. Are you sure "getHologramIcon" doesn't turn the sprites blue?

**Comment by Sligneris** [comment]
It uses the small 1x1 sprite.

File: code/modules/mob/living/silicon/ai/ai.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/modules/mob/living/silicon/ai/ai.dm]: it doesn't i'm pretty sure?

Comment (RemieRichards) [inline code/modules/mob/living/silicon/ai/ai.dm]:
```
/proc/getHologramIcon(icon/A, safety=1)//If safety is on, a new icon is not created.
	var/icon/flat_icon = safety ? A : new(A)//Has to be a new icon to not constantly change the same icon.
	flat_icon.ColorTone(rgb(125,180,225))//Let's make it bluish.
	flat_icon.ChangeOpacity(0.5)//Make it half transparent.
	var/icon/alpha_mask = new('icons/effects/effects.dmi', "scanline")//Scanline effect.
	flat_icon.AddAlphaMask(alpha_mask)//Finally, let's mix in a distortion effect.
	return flat_icon
```

LETS MAKE IT BLUEISH

**Comment by WJohn** [comment]
wouldve been funny if the ai could billow up into this gigantic alien queen projection, though.

**Pull request merged**